reverse_script.py

```diff
@@ -0,0 +1,64 @@
+import os, sys
+import re
+import json
+
+# DESCRIPTION
+# We are launching script from the command line 
+# Taking the name of the file with the dictionary (json) from the command
+# Reversing dictionary
+# Writing the result to the file (the name of the file present in system variable)
+
+# reverting dictionary (changing keys with values)
+def reversing_dict(dictionary):
+    result_dict = {}
+    buffer = []
+    for key, value in dictionary.items():
+        if value not in result_dict.keys(): #adding new entry if it is a new key

```

Comment:
Add space after #

Reply:
fixed

Reply:
Specify exact commit after "fixed ..." so that reviewer could easily validate your fix